cmd/relayer/status/status.go

```diff
@@ -41,6 +42,13 @@ func Cmd() *cobra.Command {
 			raData := rlyCfg.RaDataFromRelayerConfig()
 			hd := rlyCfg.HubDataFromRelayerConfig()
 
+			_, err = rollapp.GetMetadataFromChain(raData.ID, *hd)
+			if err != nil {
+				pterm.Error.Println("failed to fetch rollapp information from hub: ", err)
+				pterm.Error.Println("the hub rpc endpoint is not responsive")
```

Comment:
this might not be the case, if we want to explicitly return an error type that's along the lines of "endpoint unavailable" and check whether the returned error is of this type with `errors.Is`

I think returning the failed to fetch error is enough as there are outputs statin unhealthy endpoint in the retry loop already